lib/jnpr/jsnapy/jsnapy.py

```diff
@@ -739,9 +768,9 @@ def extract_data(
                     None,
                     None,
                     action)
-            if host.__contains__('include'):
+            if host.__contains__('include') or len(config_data.get('hosts'))>1:
```

Comment:
As you are checking for len > 1, did you test with only one host entry. Hope it wont break things for just one host.